dokka-integration-tests/gradle/README.md

```diff
@@ -8,21 +8,6 @@ locally (`./gradlew publishToMavenLocal`).
 
 To debug Gradle tests, the environment variable `ENABLE_DEBUG=true` should be declared.
 
-### To update git submodules 
-
-Integration tests have fixed git revision number, with the diff patch applied from the corresponding file (e.g. [`coroutines.diff`](projects/coroutines/coroutines.diff)).
```

Comment:
While having outdated documentation could be worse than having none at all, it'd be nice to describe the current process of updating the versions of external IT projects here, similarly to how it's done in this block. 

Some context: the process of updating the tested coroutines/serialization projects isn't straightforward, you have to know not only how these tests work, but also where to look for these hardcoded values and diffs. While the members of the Dokka team now how to do all that, there are cases when people outside of the Dokka team need to update these (as Dokka and its tests are sometimes used for reproducing bugs in other projects), and in such cases it's convenient to point people to an instruction of some sort, like this one